src/main/java/org/folio/service/finance/transaction/TransactionSummariesService.java

```diff
@@ -61,16 +62,18 @@ public TransactionSummariesService(RestClient restClient) {
       String orderId = Stream.concat(orderIdFromCreate, orderIdFromStorage)
         .findFirst()
         .orElse(null);
-        if (CollectionUtils.isEmpty(holder.getEncumbrancesFromStorage())) {
+      return getOrderTransactionSummary(orderId, requestContext)
+        .exceptionally(t -> null)
```

Comment:
I beleive it was expected that exception will be raised only on 404 http code. But it seems creation of new transaction summary will bee procceded on any error, despite the http code

Reply:
I added code to only catch a 404, is that what you meant ? Not catching the other issues could also potentially have side effects, since we were not getting the summary there previously.

Reply:
Actually, I see this creates issues in the mod-orders integration tests because of the missing permission.

Reply:
ah, so it looks like permission issue. thanks for pointing on this